gym/wrappers/monitoring.py

```diff
@@ -0,0 +1,41 @@
+from gym import monitoring
+from gym import Wrapper
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+def Monitor(directory, video_callable=None, force=False, resume=False,
```

Comment:
I'd prefer not to do this in decorator style, but instead have the Monitor class __init__ method takes these as arguments directly.